UnitTests/UnitTests.XamlIslands.UWPApp/FakeTestClasses/Assert.cs

```diff
@@ -3,16 +3,38 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using System.Linq;
+using Windows.UI.Xaml;
```

Comment:
Do we need this namespace?

Reply:
Ah, when I generated the method it had the Windows type in there instead of the generic I changed it to after, it can be removed.